src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
This PR is coming from a branch of your fork, cholesky_improvement, rather than master of your fork (which is good, PRs _should_ come from branches). So the state of your fork master doesn't matter at all. As I see it, the last non-PR-commit of your branch is still 17 May, so no change from my last message. But perhaps you haven't pushed back to github. So just let me know when you're ready for me to reassess, thanks!